setup.cfg

```diff
@@ -27,6 +27,8 @@ install_requires =
     statsmodels
     requests
     numba
+    seaborn
+    Bio
```

Comment:
**question:** Isn't the package name is `biopython`?

Reply:
Yes! Weird that it didn't throw an error when I tested it.